api/swagger-spec/resourceListing.json

```diff
@@ -24,6 +24,14 @@
    {
     "path": "/apis/extensions",
     "description": "get information of a group"
+   },
+   {
+    "path": "/apis/autoscaling/v1",
```

Comment:
I think i had pointed this before, but cant find that comment now.
You need to add files for these new paths.

You need to fix hack/update-swagger-spec to curl these new paths and store the files.
And then fix verify-swagger-spec as well.